cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -381,6 +381,17 @@ public static String getZKQuorum(CConfiguration cConf) {
     public static final String CONTAINER_PRIORITY_CLASS_NAME = "preview.runner.container.priority.class.name";
   }
 
+  /**
+   * Spark on k8s
+   */
+  public static final class SparkOnK8S {
```

Comment:
nit: I would just name this SparkDriver (or maybe have a Driver class within the Spark class). That way it's a bit more apparent what it is when reading the code (`Spark.Driver.ADDRESS` vs `SparkOnK8S.ADDRESS`)

Reply:
Great point. Fixed it.